dll/win32/shlwapi/utils.cpp

```diff
@@ -115,3 +133,99 @@ PathFileExistsDefExtAndAttributesW(
 
     return TRUE;
 }
+
+static inline BOOL
+SHLWAPI_IsBogusHRESULT(HRESULT hr)
```

Comment:
Not sure this is worth an inline function if you intend to use that one only once (in `IShellFolder_GetDisplayNameOf()`)

Reply:
This function is called from `IShellFolder_GetDisplayNameOf` multiple times.

Reply:
Ah, you mean it's used more than once (in Windows), and you expect to do the same thing in ReactOS as well later on? hmhm ok maybe why not then...

Reply:
Yes, I do.